src/components/views/settings/tabs/user/AppearanceUserSettingsTab.js

```diff
@@ -0,0 +1,243 @@
+/*
+Copyright 2019 New Vector Ltd
```

Comment:
This file is created in 2020, but we need to maintain the copyright for the copy/pasted code:
```suggestion
Copyright 2019 New Vector Ltd
Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
```